### PR TITLE
Translation updates for cloning and automated workflows

### PR DESCRIPTION
**Pull request by joshsadam**
## What does this PR do and why?
_Describe in detail what your merge request does and why._

Update translations for:

1. `Clone samples` --> `Copy samples`
2. `Automated Workflow Executions` --> `Automated Pipelines`

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [x] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -92,11 +92,11 @@ en:
         revoke?: You are not authorized to revoke personal access tokens for user %{name} on this server.
         update?: You are not authorized to update user %{name} on this server.
       workflow_execution:
-        cancel?: You are not authorized to cancel workflow executions for %{namespace_type} %{name}
-        create?: You are not authorized to submit workflow executions for %{namespace_type} %{name}
-        destroy?: You are not authorized to destroy workflow executions for %{namespace_type} %{name}
-        index?: You are not authorized to view workflow executions for %{namespace_type} %{name}
-        read?: You are not authorized to view workflow execution %{id}
+        cancel?: You are not authorized to cancel pipeline executions for %{namespace_type} %{name}
```
Comment (ericenns) [inline config/locales/en.yml:95]: **nitpick (blocking):** The following 4 translations should be reverted to `workflow executions`

File: config/locales/en.yml [hunk 2]
```diff
@@ -263,13 +263,13 @@ en:
...
+        outputs_and_metadata_written_html: Project automation bot attached pipeline %{href} outputs and wrote metadata to sample %{sample_href}
+        outputs_written_html: Project automation bot attached pipeline %{href} outputs to sample %{sample_href}
       automated_workflow:
-        create_html: "%{user} setup automated workflow execution %{href} for the project"
-        launch_html: Project automation bot launched workflow %{href} on sample %{sample_href}
-      automated_workflow_completion:
-        metadata_written_html: Project automation bot wrote workflow %{href} metadata to sample %{sample_href}
-        outputs_and_metadata_written_html: Project automation bot attached workflow %{href} outputs and wrote metadata to sample %{sample_href}
-        outputs_written_html: Project automation bot attached workflow %{href} outputs to sample %{sample_href}
+        create_html: "%{user} setup automated pipeline execution %{href} for the project"
```
Comment (ericenns) [inline config/locales/en.yml:271]: **nitpick (blocking):** I think you can remove the word `execution` from this translation.

File: config/locales/en.yml [hunk 3]
```diff
@@ -382,11 +382,11 @@ en:
         success: Successfully updated group %{namespace_name} with group %{group_name} link %{param_name}
     workflow_execution_actions:
       cancel:
-        error: Could not cancel workflow %{workflow_name}
-        success: Workflow %{workflow_name} was successfully canceled
+        error: Could not cancel pipeline %{workflow_name}
```
Comment (ericenns) [inline config/locales/en.yml:385]: **nitpick (blocking):** The following 4 changes should be reverted.

File: config/locales/en.yml [hunk 4]
```diff
@@ -462,12 +462,12 @@ en:
       submit_button: Submit button
     new_analysis_export_dialog:
       description:
-        plural: This export will contain COUNT_PLACEHOLDER workflow executions.
-        singular: This export will contain 1 workflow execution.
-        zero: This export will contain no workflow executions.
+        plural: This export will contain COUNT_PLACEHOLDER pipeline executions.
```
Comment (ericenns) [inline config/locales/en.yml:465]: **nitpick (blocking):** Can you revert the following 7 translation changes

File: config/locales/en.yml [hunk 5]
```diff
@@ -963,36 +963,36 @@ en:
...
+          title: No automated pipelines
         headers:
           actions: Actions
           created_at: Created
           id: ID
           name: Name
           updated_at: Updated
-          workflow_name: Workflow Name
-          workflow_version: Workflow Version
+          workflow_name: Pipeline Name
```
Comment (ericenns) [inline config/locales/en.yml:994]: **nitpick (blocking):** I think the following 2 translations can be reverted

File: config/locales/en.yml [hunk 6]
```diff
@@ -1410,7 +1410,7 @@ en:
       members: Members
       samples: Samples
       settings: Settings
-      workflow_executions: Workflow Executions
+      workflow_executions: Pipeline Executions
```
Comment (ericenns) [inline config/locales/en.yml:1413]: **nitpick (blocking):** Can you revert the following 5 translations.

File: config/locales/en.yml [hunk 7]
```diff
@@ -1671,13 +1671,13 @@ en:
       workflow_version: Workflow Version
     table:
       empty:
-        description: There are no workflow executions associated with this submitter.
-        title: No workflow executions.
+        description: There are no pipelines associated with this submitter.
```
Comment (ericenns) [inline config/locales/en.yml:1674]: **nitpick (blocking):** Can you revert the following 2 translation changes.

**Review by ericenns — CHANGES_REQUESTED** [review]
Looking good, except that more translations were changed then necessary

File: config/locales/en.yml (hunk 2 above)
Comment (joshsadam) [reply config/locales/en.yml]: Updated in 51f582d

File: config/locales/en.yml (hunk 3 above)
Comment (joshsadam) [reply config/locales/en.yml]: Updated in 51f582d

File: config/locales/en.yml (hunk 4 above)
Comment (joshsadam) [reply config/locales/en.yml]: Updated in 51f582d

File: config/locales/en.yml (hunk 5 above)
Comment (joshsadam) [reply config/locales/en.yml]: Updated in 51f582d

File: config/locales/en.yml (hunk 6 above)
Comment (joshsadam) [reply config/locales/en.yml:1416]: Updated in 51f582d

File: config/locales/en.yml (hunk 7 above)
Comment (joshsadam) [reply config/locales/en.yml]: Updated in 51f582d

**Review by ericenns — APPROVED** [review]
Looks good to me!

**Pull request merged**